yescrypt/src/lib.rs

```diff
@@ -161,6 +161,7 @@ pub fn yescrypt_kdf(
     dst
 }
 
+#[allow(unsafe_op_in_unsafe_fn)]
```

Comment:
You already allowed the lint for the whole crate.

Reply:
I did not? I enabled it in the `yescrypt::common` and `yescrypt::salsa20` modules, but otherwise it's done for each function

Reply:
Ah, true. I probably mixed `lib.rs` with a module. I think we can `allow` the lint for the whole crate in this particular case.